.requirements-pyside.txt

```diff
@@ -0,0 +1,4 @@
+# this is a seperate file, referenced in requirements-optional.txt,
+# because pip does not accept the necessary options per-requirement
+--only-binary --no-index --find-links https://pymor.github.io/wheels/
```

Comment:
Doesn't work for me with both 3.4 and a self-built 3.5. 2.7 is working though. Also, jessie ships pip 1.5.6 which does not understand '--only-binary'.